notifications_utils/polygons.py

```diff
@@ -91,12 +113,20 @@ def bounds(self):
         all_min_x, all_min_y, all_max_x, all_max_y = zip(*(
             polygon.bounds for polygon in self
         ))
+
+        min_x_wgs84, min_y_wgs84 = utm29n_to_wgs84.transform(
+            min(all_min_x), min(all_min_y),
+        )
+        max_x_wgs84, max_y_wgs84 = utm29n_to_wgs84.transform(
+            max(all_max_x), max(all_max_y),
+        )
+
         return (
-            min(all_min_x), min(all_min_y), max(all_max_x), max(all_max_y),
+            min_x_wgs84, min_y_wgs84, max_x_wgs84, max_y_wgs84,
         )
 
     @cached_property
-    def buffer_outward_in_degrees(self):
+    def buffer_outward_in_m(self):
```

Comment:
the docstring a line below needs updating from `degrees` to `metres`

Reply:
Good spot, I’ve fixed this as squashed it into 2da2fa78cd570fd86a46f021853021abcbe5788b